docs/developer-guildelines.md

```diff
@@ -28,11 +28,3 @@ where username/reponame@branch is your forked repo. Remember to checkin __node_m
 ## PR for `main` branch
 
 Once code is tested and unit tests are added, please raise a PR against the actual repo’s main branch. Note that __node_modules__ should __not__ be pushed while raising PR against __main__ branch.
-
-## PR for `releases` branch
```

Comment:
Instead of removing this releases section in developer guidance, the pointer to the releases markdown can be given here. So it becomes sharable in future as well(covering E2E guidance ).

Reply:
updated